main.c

```diff
@@ -147,17 +163,28 @@ int main(void)
     ASSERT_SUCCESS_PSA(psa_crypto_init());
 
     atecc608a_print_locked_zones();
+
     /* Verify that the device has a locked config before doing anything */
     ASSERT_SUCCESS_PSA(atecc608a_check_config_locked());
 
     ASSERT_SUCCESS_PSA(atecc608a_drv_info.p_key_management->p_export(
                          atecc608a_key_slot_device, pubkey, sizeof(pubkey),
                          &pubkey_len));
 
+    /* Import a subset of the key, omitting the 0x04 prefix - only raw X & Y */
```

Comment:
I think the driver should cut off the 0x4, not the example application, and not Mbed Crypto itself before calling the driver. Hardware may differ in how it represents keys or signatures or anything else, and it's the driver's job to abstract those differences away.

Reply:
Done!